master/buildbot/db/buildsets.py

```diff
@@ -56,8 +57,7 @@ def thd(conn):
                 if 'priority' in properties:
                     priority_property = properties.get('priority')[0]
                     priority = priority_property if priority_property \
-                                                    and priority_property.isdigit() \
-                                                    and int(priority_property) > 0 else 0
+                                                    and int(priority_property) > 0 else Priority.Default
```

Comment:
Maybe we need to detect if priority_property is a digit to stop failures coming from random chars